gazelle/python/testdata/annotation_include_dep/test.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
```

Comment:
Could you please update all the new license headers you are adding?

```suggestion
# Copyright 2024 The Bazel Authors. All rights reserved.
```

Reply:
Done!